src/sentry/api/bases/organization_events.py

```diff
@@ -107,10 +107,17 @@ def quantize_date_params(self, request, params):
     def handle_query_errors(self):
         try:
             yield
-        except (discover.InvalidSearchQuery, snuba.QueryOutsideRetentionError) as error:
+        except discover.InvalidSearchQuery as error:
+            message = six.text_type(error)
+            sentry_sdk.set_tag("query_error.reason", message)
+            raise ParseError(detail=message)
+        except snuba.QueryOutsideRetentionError as error:
+            sentry_sdk.set_tag("query_error.reason", "QueryOutsideRetentionError")
             raise ParseError(detail=six.text_type(error))
```

Comment:
Do we still need `six` here?

Reply:
Black was still complaining when I flipped to `str` :(

Reply:
Wondering if black needs to be updated.

Reply:
We ignored this rule so I've removed references to six from this file 🥳